app/login/page.tsx

```diff
@@ -59,10 +61,11 @@ export default function Login({
               Join Gridiron Survivor
             </h1>
             <p className="pb-4 font-normal leading-7 text-zinc-500">
-              <Link href="/register">
-                Log in to your existing account or sign up to get started with a
-                league
-              </Link>
+              Log in to your existing account or{' '}
+              <Link href="/register" className="underline">
```

Comment:
Why do we have to add the underline class? Why is this text link not a traditional hyperlink?

Reply:
This `Link` is using `Next/Link`. I noticed I had to add the underline to show that it was link so users can click on it.

I can switch it to an `<a href> </a>` if preferred.

If I had to chose between using `Next/Link` or `<a href> </a>` , I'd prefer to use the `Next/Link` since we are using Next.

Thoughts?